markdown/ch4.markdown

```diff
@@ -291,7 +290,7 @@ freqs, times, Sx = signal.spectrogram(audio, fs=rate, window='hanning',
                                       detrend=False, scaling='spectrum')
 
 ## Plot using:
-# plt.pcolormesh(times, freqs, 10 * np.log10(Sx), cmap='viridis');
+# plt.pcolormesh(times, freqs, 10 * np.log10(Sx));
```

Comment:
Because I don't want to plot the colormesh again (already did just above using our calculation).